docker/Dockerfile

```diff
@@ -2,37 +2,45 @@ FROM ubuntu:xenial
 
 VOLUME ["/var/www/html"]
 
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
 # Install required Ubuntu packages for having Apache PHP as a module
 # as well a bunch of other packages
-RUN apt-get update && apt-get install -y \
-	apache2 \
-	composer \
-	curl \
-	less \
-	libapache2-mod-php7.0 \
-	mysql-client \
-	nano \
-	php-apcu \
-	php-xdebug \
-	php7.0 \
-	php7.0-cli \
-	php7.0-curl \
-	php7.0-gd \
-	php7.0-json \
-	php7.0-ldap \
-	php7.0-mbstring \
-	php7.0-mysql \
-	php7.0-opcache \
-	php7.0-pgsql \
-	php7.0-soap \
-	php7.0-sqlite3 \
-	php7.0-xml \
-	php7.0-xsl \
-	php7.0-zip \
-	ssmtp \
-	subversion \
-	sudo \
-	vim \
+RUN \
+	apt-get update \
+	&& apt-get install -y language-pack-en-base software-properties-common \
```

Comment:
These were required for having `add-apt-repository` and ensuring onderj's repo behaves with UTF8.